src/backend/gl/gl_common.c

```diff
@@ -47,6 +47,22 @@ struct gl_blur_context {
 	int npasses;
 };
 
+/*
+ * @brief Calculates next closest power of two of 32bit integer n
+ * ref: https://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2
+ */
+int NextPowerOfTwo(int n)
```

Comment:
Can you put this in `utils.h` or `utils.c`? Also the name should be `next_power_of_two`.

`glGetUniformLocationChecked` below is an exception since it's meant to look like an OpenGL function.